util/util.go

```diff
@@ -491,3 +493,13 @@ func truncatePosition(position token.Position) token.Position {
 func PosToLocation(pos token.Pos, pass *analysis.Pass) token.Position {
 	return truncatePosition(pass.Fset.Position(pos))
 }
+
+// ExprToString converts AST expression to string
+func ExprToString(e ast.Expr, pass *analysis.Pass) string {
+	var buf bytes.Buffer
+	err := printer.Fprint(&buf, pass.Fset, e)
+	if err != nil {
+		panic(fmt.Sprintf("Failed to convert AST expression to string: %v\n", err))
+	}
+	return buf.String()
+}
```

Comment:
I'm having dejavu of this, didn't we add this in some other PRs?

Anyways, not something to fix :)

Reply:
Haha, yes. This function was introduced in the revised error message format PR at one point, but removed later since it did not have any use. So, it was re-added in this PR since we want to convert LHS and RHS `ast.Expr` in an assignment statement to string.